fastlane/metadata/android/en-US/short_description.txt

```diff
@@ -0,0 +1 @@
+Manage local files
```

Comment:
This short description is incorrect. Amaze also handles remote files on ftp / smb servers, I think a simply 
`Light and smooth file manager following the Material Design guidelines` should be okay.

Reply:
Looks like it's a very old description. I'll only keep the en-US and zh-CN descriptions since I can understand them.